index.js

```diff
@@ -2,6 +2,7 @@
 
 module.exports = {
   rules: {
-    'accessing-ember-namespace': require('./lib/rules/accessing-ember-namespace')
+    'accessing-ember-namespace': require('./lib/rules/accessing-ember-namespace'),
+    'disallow-function-prototype-extension': require('./lib/rules/disallow-function-prototype-extension')
```

Comment:
The descriptions should be consistently describing enforced behavior, or disallowed behavior. For example `accessing-ember-namespace` describes something disallowed, so this new description likely does not need `disallow-`. All descriptions are of disallowed behavior.